docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -336,5 +336,12 @@ is adjusted to avoid a bit shift that is greater than the type length:
      ...
    }
 
-
-
+**How do I resolve missing include errors that occur when migrating my
+ code?

```

Comment:
The title for the FAQ entry should be on a single line with ** at the beginning and end, for example **my title**